_includes/v20.2/backups/backup-options.md

```diff
@@ -2,5 +2,5 @@
 -----------------------------------------------------------------+-------------------------+------------------------------
```

Comment:
As we mentioned in the meeting could you add the changes to the restore docs in the same way?

Reply:
added restore v20.2-22.2